src/Logging/src/DynamicSerilogBase/Steeltoe.Extensions.Logging.DynamicSerilogBase.csproj

```diff
@@ -1,17 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
-    <Description>Steeltoe library for enabling dynamic management of Serilog</Description>
-    <PackageTags>logging;dynamic logging;serilog;aspnetcore;management;monitoring;Spring Cloud;</PackageTags>
+    <Description>Steeltoe library for enabling dynamic management of Serilog. Includes Console sink.</Description>
```

Comment:
Added note about the Console sink to the description... If we remove the reference to Serilog.Sinks.Console, the packages still built but 3 tests fail... I could have moved the reference to the tests, but I _think_ console logging is used in most/all Steeltoe use cases, so it probably makes sense to leave it here and just call it out